tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -1111,7 +1164,9 @@ private void resolveVotes() {
         winningStatus = computeStatus(weights[0], weights[1]);
 
         // if we are not as good as the baseline (trunk), use the baseline
+        // TODO: how could baselineStatus be null here??
```

Comment:
maybe a newly-added (and/or "extra"???) path with no baseline value/status? or maybe it's impossible -- better safe than sorry though

Reply:
It was checked above